### PR TITLE
Peer code review

### PR DESCRIPTION
**Pull request by amiraabouhadid**
## Description

In this project, a tasks list structure was generated using javascript bundledthrough webpack. 

### General requirements

- [x] There are no linter errors.
- [x] Followed the **Github flow**.
- [x] The documentation is professional.

### HTML/CSS/JS requirements

- [x] We followed the list of [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
- [x] We followed the list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).

### Project requirements

- Add a new JavaScript file and import it as a module:
- it will contain methods related to the status updates (completed: true / false).
- Add event listener to the checkbox (change).
- Update items object's value for completed key upon user actions.
- Implement a function for the "Clear all completed" button (use filter() method).
- Store the updated array of items in local storage, so the user gets the correct list values after the page reloads.

Thank you for taking the time to review this pull request.

### CODE REVIEW

File: src/modules/Home.js [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+        " d-flex border p-3 container justify-content-between align-items-baseline bg-white position-absolute  ";
+      const taskContainerDesc = document.createElement("div");
+      taskContainerDesc.classList =
+        "d-flex align-items-baseline justify-content-between ";
+      const incompleteTaskCheck = document.createElement("input");
+      incompleteTaskCheck.type = "checkbox";
+      incompleteTaskCheck.onclick = (e) => {
+        e.preventDefault();
+        if (task.complete === true) {
+          //true
```
Comment (zmekonnen251) [inline src/modules/Home.js]: - It is good to use conditional shorthands. you don't need to check it `if(task.complete)` is enough.

File: src/modules/Home.js [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+          }
+        });
+        localStorage.setItem("tasks", JSON.stringify(tasks));
+
+        this.displayHome();
+      };
+      incompleteTaskCheck.style.marginRight = "1rem";
+      const incompleteTaskDescription = document.createElement("p");
+      incompleteTaskDescription.innerHTML = task.description;
+      if (task.complete == true) {
```
Comment (zmekonnen251) [inline src/modules/Home.js:150]: - It is good to use conditional shorthands. you don't need to check it `if(task.complete)` is enough.

**Review by zmekonnen251 — CHANGES_REQUESTED** [review]
# CHANGES REQUIRED 
Hello @amiraabouhadid  great job you are doing so far in this project. 💯

TO HIGHLIGHT:

- Your project is professional 
- Your repo is professional 
- Well structured files 

However, there are a few changes to make to move to the next project 🛠️ . Keep working hard 💪 💻 

## Required Changes ♻️
See my inline comment to know what changes are needed. 👇 👀 

Please make these required changes and send it again for another review 👍 💪
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
# HAPPY CODING!!! 😄 

cat


**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

--------

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/modules/Home.js (hunk 1 above)
Comment (amiraabouhadid) [reply src/modules/Home.js]: Thanks for pointing this out, I believe it is fixed now.

File: src/modules/Home.js (hunk 2 above)
Comment (amiraabouhadid) [reply src/modules/Home.js:150]: Thanks for pointing this out, I believe it is fixed now.

**Review by zmekonnen251 — APPROVED** [review]
Hi @amiraabouhadid ,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------
>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**